modules/ROOT/pages/ldap/ldap-connector-reference.adoc

```diff
@@ -504,7 +505,7 @@ When you know the DN of the object you want to retrieve, use this operation:
 | Configuration | String | The name of the configuration to use. | | x
 | DN a| String |  The DN of the LDAP entry to retrieve. |  | x
 | Attributes a| Array of String |  A list of the attributes to return in the result. If the attributes list is empty or null, then by default all LDAP entry attributes are returned. |  |
-| Structural Object Class a| String |  The type of entry to return. Only for DataSense purposes to be used in Anypoint Studio IDE. Has no impact on runtime, that's why it's optional. |  |
+| Structural Object Class a| String |  The type of entry to return. Only for DataSense purposes to use in Anypoint Studio IDE. Has no impact on runtime, that's why it's optional. |  |
```

Comment:
There are some issues with this text but I am not really sure what it is trying to say. What is IDE? I think you should spell it out: "xxx (IDE)"

Here is my suggestion, but I am not sure if the meaning is right:

The type of entry to return. This is used for DataSense in Anypoint Studio IDE and has no impact on Mule.